std/typecons.d

```diff
@@ -3675,6 +3675,59 @@ struct Nullable(T)
 
     private bool _isNull = true;
 
+    /**
+     * Compares two Nullable values.
+     * - If one is null and the other is not, the null one is considered smaller.
+     * - If both are null, they are equal.
+     * - If both are non-null, compares the payloads.
+     *
+     * Returns:
+     *     Negative if `this < rhs`, zero if equal, positive if `this > rhs`.
+     */
+    int opCmp(this This, Rhs)(auto ref Rhs rhs) const
```

Comment:
Use ``typeof(this)`` instead of ``this This`` if you need the this pointers type.

Reply:
Since it is no longer needed, you can remove this template parameter.

Reply:
```suggestion
    int opCmp(Rhs)(auto ref Rhs rhs) const
```